premake5.lua

```
@@ -0,0 +1,139 @@
+        
+    -- Configuration-specific settings
+    filter "configurations:Debug"
+        defines { "DEBUG" }
+        symbols "On"
+
+    filter "configurations:Release"
+        defines { "NDEBUG" }
+        optimize "On"
+
+-- Create projects for each example file that matches the pattern
+for _, filepath in ipairs(getExampleFiles()) do
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'getExampleFiles'

(W113)